`Development`: Add documentation for bonus, publishing example solutions, plagiarism checks for exams and special grades (#6087)
ls1intum/Artemis: docs/user/exams/instructors_guide.rst

    Unsubmitted exams are not eligible for the assessment process.

- Review student performance using various metrics such as average, median and standard deviation.

> - Unsubmitted exams are not eligible for assessment and thereby appear as having no score and the corresponding students are assigned with :ref:`no-participation special grade <grading>` if a grading key exists. It can happen that an exercise is not part of any :ref:`student exam <student_exams>`. This is the case when Artemis selects a different exercise of the same exercise group for every :ref:`student exam <student_exams>`. Similarly to the unsubmitted exams, they can warp the results and statistics of the exam. By eliminating unsubmitted exams and exercises which were not part of the :ref:`exam conduction <exam_conduction>`, you can gain a more realistic overview of the performance of the students.

Reviewer: ```suggestion
- Unsubmitted exams are not eligible for assessment and thereby appear as having no score. The corresponding students are assigned with a :ref:`no-participation special grade <grading>` if a grading key exists. It can happen that an exercise is not part of any :ref:`student exam <student_exams>`. This is the case when Artemis selects a different exercise of the same exercise group for every :ref:`student exam <student_exams>`. Similarly to the unsubmitted exams, they can warp the results and statistics of the exam. By eliminating unsubmitted exams and exercises which were not part of the :ref:`exam conduction <exam_conduction>`, you can gain a more realistic overview of the performance of the students.
```

   Exam Scores Page

4.2. Exam Solutions
^^^^^^^^^^^^^^^^^^^

> Optionally, you can publish the example solutions of text, modeling, file upload and programming exercises to the participated students after a desired date by setting ``Example Solution Publication Date`` of the exam to a non-empty date.

Reviewer 2: ```suggestion
Optionally, you can publish the example solutions of text, modeling, file upload and programming exercises to the registered students after a desired date by setting ``Example Solution Publication Date`` of the exam to a non-empty date.
```
participated seams off here. If "registered" is not correct, you should find some other wording.

Reviewer 3: I think this sounds the best.
```suggestion
Optionally, you can publish the example solutions of text, modeling, file upload and programming exercises to students with submissions after a desired date by setting ``Example Solution Publication Date`` of the exam to a non-empty date.
```

> All exercises in the exam with the given types are published according to this date, as opposed to the course exercises which have their own individual example solution publication dates.

Reviewer 2: Aren't the example solutions published?
And what types are you referring to?

Author: With `type`, I was referring to text, modeling, file upload and programming exercises. I can change the wording if it is not clear, do you have any suggestions?

Reviewer 2: ```suggestion
All example solutions of these exercises types are published according to this date set in the exam, as opposed to the course exercises which have their own individual example solution publication dates.
```
